test/e2e/v1beta1/scripts/gh-actions/build-load.sh

```diff
@@ -62,6 +63,11 @@ _install_tools() {
   fi
 }
 
+setup_for_tune() {
+  docker pull $DEFAULT_IMAGE_FOR_TUNE
+  _build_containers "suggestion-hyperopt" "$CMD_PREFIX/suggestion/hyperopt/$VERSION/Dockerfile"
```

Comment:
Do we need this since we replace images for all suggestion during `run()` script ?

Reply:
Yeah, I think we need it. 

In the test case of `tune`, we do not specify any `EXPERIMENTS`(`EXPERIMENTS_ARRAY`). So the `suggetions` is empty and scripts file will build no containers in CI/CD env. We need to build `random` suggestion image for `tune` function. https://github.com/kubeflow/katib/blob/51b246fa1c23db130c373fa9749897a0d0b334ab/test/e2e/v1beta1/scripts/gh-actions/build-load.sh#L72-L95

Reply:
I see, @tenzen-y Do you remember is there any specific reason why we are only replacing images for Experiments' algorithms and not for all supported algorithms?

Reply:
As I commented in another thread, I believe that we can avoid this unnecessary image-building by reusing workflows.